eng/testing/debug-dump-template.md

```diff
@@ -1,127 +1,147 @@
-# Debugging a CI dump
+# Get the dump
 
-This document describes how to debug a CI/PR test dump by downloading assets from helix, using a dotnet tool called `runfo`.
+Click the link to the dump on the `Helix Test Logs` tab in Azure DevOps. This is the same place you got these instructions from.
 
-## What is runfo?
+# Get the Helix payload
 
-Runfo is a dotnet global tool that helps get information about helix test runs and azure devops builds. For more information see [this](https://github.com/jaredpar/runfo/tree/master/runfo#runfo)
-
-### How do I install it?
-
-You just need to run:
-
-```script
+[Runfo](https://github.com/jaredpar/runfo/tree/master/runfo#runfo) helps get information about helix test runs and azure devops builds. We will use it to download the payload and symbols:
+```sh
 dotnet tool install --global runfo
-```
-
-If you already have it installed, make sure you have at least version `0.6.1` installed, which contains support to download helix payloads. If you don't have the latest version just run:
-
-```script
 dotnet tool update --global runfo
 ```
-
-## Download helix payload containing symbols:
-
-You can just achieve this by running:
-
-```script
-runfo get-helix-payload -j %JOBID% -w %WORKITEM% -o <out-dir>
+If prompted, open a new command prompt to pick up the updated PATH.
+```sh
+# On Windows
+# assumes %WOUTDIR% does not exist
+runfo get-helix-payload -j %JOBID% -w %WORKITEM% -o %WOUTDIR%
+# On Linux and macOS
+# assumes %LOUTDIR% does not exist
+runfo get-helix-payload -j %JOBID% -w %WORKITEM% -o %LOUTDIR%
 ```
 
 > NOTE: if the helix job is an internal job, you need to pass down a [helix authentication token](https://helix.dot.net/Account/Tokens) using the `--helix-token` argument.
 
-This will download the workitem contents under `<out-dir>\workitems\` and the correlation payload under: `<out-dir>\correlation-payload\`. 
+Now extract the files:
 
-> The correlation payload is usually the testhost or core root, which contain the runtime and dotnet host that we use to run tests.
+```sh
+# On Windows
+for /f %i in ('dir /s/b %WOUTDIR%\*zip') do tar -xf %i -C %WOUTDIR%
 
-Once you have those assets, you will need to extract the testhost or core root. Then extract the workitem assets into the same location where coreclr binary is.
+# On Linux and macOS
+# obtain `unzip` if necessary; eg `sudo apt-get install unzip` or `sudo dnf install unzip`
+find %LOUTDIR% -name '*zip' -exec unzip -d %LOUTDIR% {} \;
+```
 
-## Windows dump on Windows
+Now use the [dotnet-sos global tool](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-sos) to install the SOS debugging extension.
+```sh
+dotnet tool install --global dotnet-sos
+dotnet tool update --global dotnet-sos
+```
+If prompted, open a new command prompt to pick up the updated PATH.
+```sh
+dotnet sos install --architecture Arm
```

Comment:
The last one will override the others, at least on Linux (not sure about windows, haven't seen if it is conditional on the bitness of the extension host, but given the errors I've seen I'd say no). Users should only install the one they are going to use.

Reply:
Oh - I didn't notice that because I was testing with x64, which is last!

Could we imagine installing them all, and picking the right one - eliminate another decision point? cc @mikem8361

Reply:
I'll fix the text meantime

Reply:
It looks like it defaults silently to the bitness of the dotnet.exe you're running it with (?)

Reply:
Yes it does default to the architecture of the dotnet.exe you are running (the most common case).

As far as installing all the of the architectures, there would have to be a rid named subdirectory to separate them so the decision point would be postponed a little to the .load command in windbg. 

This manual dotnet-sos install/load on Windows was really meant as a fallback while we wait for the public debugger to include SOS (all architectures) like the internal one does now.  The internal Windows debugger automatically loads the correct architecture and latest version of SOS from the extension gallery.

Reply:
Oh perfect then it will go away.